### ISSUE TITLE
BigInt is on Node.js 10.10 and Chrome 69+

### ISSUE THREAD

**Issue by amejiarosario** [eslint/eslint#10860]
**Tell us about your environment**

* **ESLint Version:** 4.19.1
* **Node Version:** 10.10.0
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
function test(){
  return 1n + 2n;
}

test();
```

```bash

```

**What did you expect to happen?**
I don't expect to have warnings

**What actually happened? Please include the actual, raw output from ESLint.**
I'm using VS Code with eslint pluging. It's anoying to get warning for valid JS.

![image](https://user-images.githubusercontent.com/418605/45580284-7b020780-b85d-11e8-9652-cfb115685594.png)

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report. `BigInt` is a [stage 3 ECMAScript feature](https://github.com/tc39/proposals/tree/eae1a28af0b2498927a82253c3d90e67316c1dfd#stage-3), so it's experimental and is not officially part of JS yet. As described in our [readme](https://github.com/eslint/eslint/tree/c5b688e5350d16b330222f13749ab8b6ead517f8#what-about-experimental-features), we don't support stage 3 features. We recommend using a custom parser such as [`babel-eslint`](https://github.com/babel/babel-eslint) if you'd like to parse experimental syntax.

**Comment by nzakas** [comment]
Looks like this question has been answered. As this is a question rather than an action item, I'm closing the issue. If you still need help, please send a message to our [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint). Thanks!
